client/src/themes/theme.js

```diff
@@ -40,5 +43,10 @@ export const theme = createMuiTheme({
         fontWeight: "bold",
```

Comment:
Don't forget that you can also add any arbitrary keys here and access them on makeStyles